crates/wasi-demo-app/src/main.rs

```diff
@@ -3,24 +3,28 @@ use std::{env, fs::File, io::prelude::*, process, thread::sleep, time::Duration}
 fn main() {
     let args: Vec<_> = env::args().collect();
     let mut cmd = "daemon";
-    if args.len() >= 2 {
-        cmd = &args[1];
+    if !args.is_empty() {
+        // temporary work around for wasmedge bug
+        // https://github.com/WasmEdge/wasmedge-rust-sdk/issues/10
+        if !(args.len() == 1 && args[0].is_empty()) {
```

Comment:
@ipuustin this is the work around for the wasmedge bug.  

I should also call out that this change set, is potentially a breaking change.  Any module that took arguments, was  throwing away the first argument as it was the module name.  We no pass arguments as a program might expect.